Packs/CommonScripts/Scripts/FindSimilarIncidentsV2/FindSimilarIncidentsV2.py

```diff
@@ -108,20 +108,31 @@ def get_incident_labels_map(labels):
 
 
 def prepare_value_to_query_with(value):
-    str_value = str(value) if isinstance(value, int) else value
-    str_value = str_value.replace('"', r'\"').replace("\n", "\\n").replace("\r", "\\r")
-    str_value = str_value.encode('utf-8') if not isinstance(value, int) else str_value
-    return str_value
+    str_value_list = []
+    value_list = value
+    if isinstance(value, list) and not value:
```

Comment:
what this condition apply for?
if it's a list and not value, it doesn't mean it's make sure it's empty list?